Monika After Story/game/script-fun-facts.rpy

```diff
@@ -788,3 +788,67 @@ label mas_fun_fact_coffee_origin:
     m 3hub "...And I for one can certainly attest that the love of coffee has remained strong to this day!"
     call mas_fun_facts_end
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_fun_facts_database,
+            eventlabel="mas_fun_fact_synesthesia",
+        ),
+        code="FFF"
+    )
+
+label mas_fun_fact_synesthesia:
+    m 1esa "Ok, this one's pretty interesting."
+    m 3eua "Some people experience a phenomenom known as {i}synesthesia{/i},{w=0.1} which is where something that stimulates one of our senses also triggers another sense simultaneously."
+    m 1hua "That's kind of a wordy explanation hehe...{w=0.2} Let's find an example!"
+    m 1eua "It says here a common form of synesthesia is {i}grapheme–color synesthesia{/i},{w=0.1} which is where people 'experience' letters and numbers as colours."
```

Comment:
```suggestion
    m 1eua "It says here that a common form of synesthesia is {i}grapheme–color synesthesia{/i},{w=0.1} which is where people 'experience' letters and numbers as colours."
```

I am not sure if MAS format uses "colours" or "colors" as well.

Reply:
I assume it would be "colors" since all the other spelling is American-English.